language/transaction-builder/generator/src/common.rs

```diff
@@ -51,8 +53,7 @@ fn quote_parameter_as_field(arg: &ArgumentABI) -> Named<Format> {
     }
 }
 
-// TODO(#7876): Update to handle script function ABIs
-pub(crate) fn make_abi_enum_container(abis: &[TransactionScriptABI]) -> ContainerFormat {
+pub(crate) fn make_abi_enum_container(abis: &[ScriptABI]) -> ContainerFormat {
```

Comment:
Ah we landed the renaming, but this is now now taking a full `ScriptABI`  (including script functions) since those are now supported, and not just `TransactionScriptABI`s if that makes sense?